src/MixpanelEventForwarder.js

```diff
@@ -245,8 +284,20 @@ var constructor = function () {
     this.process = processEvent;
     this.setUserAttribute = setUserAttribute;
     this.setUserIdentity = setUserIdentity;
-    this.onUserIdentified = onUserIdentified;
     this.removeUserAttribute = removeUserAttribute;
+
+    this.onLoginComplete = onLoginComplete;
+
+    // For all Identity Requests, we run mixpanel.identify(<device_id)
```

Comment:
`device_id` should be `user_id` right?  We never manually pas in `device_id` anywhere

```suggestion
    // For all Identity Requests, we run mixpanel.identify(<device_id)
```